src/main.ts

```diff
@@ -18,7 +18,9 @@ async function run(): Promise<void> {
     core.startGroup(`check: ${check.name}`)
     try {
       if (await triggeredByBot()) {
-        core.info('Action triggered by bot, skipping checks')
+        core.info('Action triggered by bot, skipping all checks')
+      } else if (checkSkipped(check)) {
```

Comment:
It would make more sense to make the check first if it's skipped by code, and then by the bot (like this might be more extendable - in the case that it makes sense to be run even for bots, but not if they are skipped)

Reply:
Right now we are always skipping every check when the change is triggered by a bot, that's why I left that condition first since it takes precedence. If we change that behavior for future checks, we can revisit

Reply:
I moved the condition for bots up so it's clearer in the code and in the output that it's skipping all checks (and no need to iterate over every check for the triggered by bot skipping logic)

Reply:
Nice, this is now reflecting better existing functionality